tests/integration/components/ancestry-display/component-test.ts

```diff
@@ -0,0 +1,112 @@
+import EmberObject from '@ember/object';
+import Service from '@ember/service';
+import { render } from '@ember/test-helpers';
+import I18N from 'ember-i18n/services/i18n';
+import { setupRenderingTest } from 'ember-qunit';
+import { TestContext } from 'ember-test-helpers';
+import faker from 'faker';
+import hbs from 'htmlbars-inline-precompile';
+import { module, test } from 'qunit';
+
+const i18nStub = Service.extend({
+    translations: EmberObject.create({
+        general: {
+            ellipsis: '...',
+        },
+    }),
+
+    t(key: string): string {
```

Comment:
If you're only ever return the ellipsis here, you can just have `t` return `'...'`.